cluster/charts/rook-ceph/values.yaml

```diff
@@ -271,6 +271,11 @@ csi:
 enableFlexDriver: false
 enableDiscoveryDaemon: false
 
+# enable the ability to have multiple ceph filesystem's in the sqame cluster
+# WARNING: Experimental feature in octopus, nautilus and mimic
```

Comment:
Nit: I would be a little more explicit about Ceph versions so newer users aren't confused
```suggestion
# WARNING: Experimental feature in Ceph Releases Octopus (v15) and Nautilus (v14)
```

Mimic isn't supported by rook, so I don't think we need to mention it.

Reply:
I've applied your suggestion, thanks for the review!